### PR TITLE
Passwordreset

### PR DESCRIPTION
**Pull request by ShawnCollinge**
Author: Shawn

## What changes were made?
Added a shitty password reset form, will style it later just need to allow people to update passswords.

### CODE REVIEW

File: src/pages/ResetPassword.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { resetPassword } from '../services/member';
+
+const ResetPasswordForm: React.FC = () => {
+  const [newPassword, setNewPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [error, setError] = useState('');
```
Comment (elimelt) [inline src/pages/ResetPassword.tsx:8]: would it be more consistent to set these as undefined initially?

**Review by elimelt — APPROVED** [review]
LGTM, just left question

File: src/pages/ResetPassword.tsx (hunk 1 above)
Comment (elimelt) [reply src/pages/ResetPassword.tsx:8]: (not all just error and success)

Comment (ShawnCollinge) [reply src/pages/ResetPassword.tsx:8]: Yeah, I'm going to redo this page later. This is a shitty version of it that chatgpt made

**Pull request merged**